### PR TITLE
OpenNebula one_vm.py: Fix missing keys

### PR DESCRIPTION
**Pull request by xorel**
##### SUMMARY
Fixes: #685

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
one_vm.py

##### ADDITIONAL INFORMATION

```
ansible_collections/community/general/plugins/modules/one_vm.py", line 727, in get_vm_info
KeyError: 'IP'

---
ok: [localhost] => {
    "msg": "result"
}

```

### CODE REVIEW

File: plugins/modules/cloud/opennebula/one_vm.py [hunk 1]
```diff
@@ -752,11 +752,19 @@ def get_vm_info(client, vm):
     if 'NIC' in vm.TEMPLATE:
         if isinstance(vm.TEMPLATE['NIC'], list):
             for nic in vm.TEMPLATE['NIC']:
-                networks_info.append({'ip': nic['IP'], 'mac': nic['MAC'], 'name': nic['NETWORK'], 'security_groups': nic['SECURITY_GROUPS']})
+                networks_info.append({
+                    'ip': vm.TEMPLATE['NIC'].get('IP',''),
```
Comment (nilsding) [inline plugins/modules/cloud/opennebula/one_vm.py:756]: shouldn't this be `nic.get('IP', '')` here?

Comment (xorel) [reply plugins/modules/cloud/opennebula/one_vm.py]: you are right, copy & paste eror

**Comment by felixfontein** [comment]
Please fix the failing sanity tests, and add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@xorel thanks for fixing this!
@nilsding thanks for reviewing!

I've just merged this since the change looks pretty safe, and so it can be included in today's bugfix release. Hope that's ok :)

**Pull request merged**